cmd/xl-storage-disk-id-check.go

```diff
@@ -579,3 +514,276 @@ func (p *xlStorageDiskIDCheck) updateStorageMetrics(s storageMetric, paths ...st
 		}
 	}
 }
+
+const (
+	diskHealthOK = iota
+	diskHealthOffline
+
+	// diskMaxConcurrent is the maximum number of running concurrent operations.
+	diskMaxConcurrent = 100
```

Comment:
```suggestion
	diskMaxConcurrent = 100
```

we have to make this configurable for testing purposes to arrive at valid values.